test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -1841,5 +1841,43 @@ public static CommandRunnerResult RunInstall(SimpleTestPathContext pathContext,
 
             return r;
         }
+
+        [Fact]
+        public void InstallCommand_NF_Project()
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                var workingPath = pathContext.WorkingDirectory;
```

Comment:
use explicit type

Reply:
I'll change all these. For my education: what's the criteria that you're following to choose `var` vs. explicit type?

Reply:
I linked our guidelines above. Do you have a question about a specific part?

Reply:
I've missed that link to the guidelines.
If I'm not mistaken I have them all replaced as requested.

A side comment: I took the var names, declarations, workflow from existing tests, basically tried to follow the existing "style". In future PRs, I'll disregard that and will try to follow the guidelines.

Reply:
> A side comment: I took the var names, declarations, workflow from existing tests, basically tried to follow the existing "style". In future PRs, I'll disregard that and will try to follow the guidelines.

Yes, unfortunately the code isn't a good example. The guidelines are what we're working toward with all new code and changes.